cloud_chatops/src/pr_reminder.py

```diff
@@ -181,12 +174,41 @@ def get_real_name(self, username: str) -> str:
             name = username
         return name
 
-    def github_to_slack_username(self, user: str) -> str:
+    def _github_to_slack_username(self, user: str) -> str:
         """
         This method checks if we have a Slack id for the GitHub user and returns it.
         :param user: GitHub username to check for
         :return: Slack ID or GitHub username
         """
-        if user in self.slack_ids:
-            return self.slack_ids[user]
+        if user not in self.slack_ids:
+            # If the PR author is not in the Slack ID mapping
+            # then we set the user to mention as David Fairbrother
+            # as the team lead to deal with this PR.
+            user = "U01JG0LKU3W"
```

Comment:
another one where it would be best to have this as a parameter to the method - something like `default_user_id` with this as the default